elasticdl/proto/elasticdl.proto

```diff
@@ -134,3 +134,12 @@ service Master {
     rpc ReportEvaluationMetrics(ReportEvaluationMetricsRequest) returns (ReportEvaluationMetricsResponse);
     rpc ReportTaskResult(ReportTaskResultRequest) returns (google.protobuf.Empty);
 }
+
+// PS service
+// TODO: define messages and replace `google.protobuf.Empty` with correct message.
+service Pserver {
```

Comment:
How about ParameterServerService?

Reply:
proto will generate a function `elasticdl_pb2_grpc.add_PserverServicer_to_server` if named `Pserver`. 
If changed to `ParameterServerService`, it will be `elasticdl_pb2_grpc.add_ParameterServerServiceServicer_to_server`.  That is too long and reduntant.

Reply:
I think xxxxService is better than `Pserver`.

Reply:
[Another issue](https://github.com/sql-machine-learning/elasticdl/issues/1338) is responsible for adding message and rpc services in the proto file.
We can modify it if needed in that PR.